### PR TITLE
Containers accessibility

### PR DESCRIPTION
**Pull request by YahelCh**

### CODE REVIEW

File: src/rules/accessibility/containersAccessibility.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    id: 'containers-accessibility',
+    description: 'div with class container should have role of tabPanel and attr binding of id and aria-labeledby',
+    init: function (parser, reporter) {
+        var self = this;
+        var isContainer = function (element, attributes) {
+            return (element === "div" && HTMLHint.utils.hasClass(attributes, "container"));
+        };
+
+        var bindingContains = function (bindings, requestedBind) {
+            return !bindings.includes('attr') || !bindings.includes(requestedBind);
```
Comment (rziv) [inline src/rules/accessibility/containersAccessibility.js]: the its unclear why the attr check is needed. Please add a comment

File: src/rules/accessibility/containersAccessibility.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        parser.addListener('tagstart', function (event) {
+            var tagName = event.tagName.toLowerCase();
+
+            if (isContainer(tagName, event.attrs)) {
+                var roleAttr = HTMLHint.utils.getAttributeValue(event.attrs, "role");
+                var bindingsAttr = HTMLHint.utils.getAttributeValue(event.attrs, "data-bind");
+                if (roleAttr !== 'tabpanel') {
+                    reporter.error('div with class container should have role "tabpanel"', event.line, event.col, self, event.raw);
+                }
+
```
Comment (rziv) [inline src/rules/accessibility/containersAccessibility.js]: the name bindingContains implies a positive check but seems to be used as a negative one

File: test/rules/accessibility/containersAccessibility.spec.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+    it('div with class "container" with no attr bind of "aria-labeledby" should result in an error', function () {
+        var code = `<div class="container" role="tabpanel" data-bind="attr:{'id':'tabpanel_+'name()}"></span>`;
+        var messages = HTMLHint.verify(code, ruleOptions);
+        expect(messages.length).to.be(1);
+        expect(messages[0].message).to.equal('div with class container should have attr binding of aria-labeledby to an id that starts with "tab_"');
+        expect(messages[0].rule.id).to.be(ruldId);
+    });
+
+    it('div with class "container" with attr bind of "aria-labeledby" that does not start with "tab_"should result in an error', function () {
```
Comment (rziv) [inline test/rules/accessibility/containersAccessibility.spec.js]: wouldn't you have a false negative if another data-bind started with "tab_" ?
for example if the html was:
var code = `<div class="container" role="tabpanel" data-bind="attr:{'someAttribute':'tab_1', 'id':'tabpanel_+'name(),aria-labeledby':name()}"></span>`;
(this situation applies to other tests as well).

**Comment by YahelCh** [comment]
After wrap the requested attr bindings in bindingHandler "addContainerAccessibilityAttrs" (see components/navigation/bindingHandlers.js) adjust rule and tests.

**Pull request closed without merging**